### PR TITLE
ping command

### PR DESCRIPTION
**Pull request by MrSam7K**
we are so ping pong

image

### CODE REVIEW

**Comment by RedVortexDev** [comment]
turn the Objects.requireNonNull check you removed and `CodeClient.MC.player != null` to a `if (CodeClient.MC.player == null || X) return;` so its cleaner and doesnt throw an error in some weird state where they are null

File: src/main/java/dev/dfonline/codeclient/Commands.java [hunk 1]
```diff
@@ -610,7 +610,7 @@ else if(field.getType().isEnum()) {
 
         dispatcher.register(literal("ping").executes(context -> {
             assert CodeClient.MC.player != null;
-            int ping = Objects.requireNonNull(CodeClient.MC.player.networkHandler.getPlayerListEntry(CodeClient.MC.player.getUuid())).getLatency();
+            int ping = CodeClient.MC.player.networkHandler.getPlayerListEntry(CodeClient.MC.player.getUuid()).getLatency();
```
Comment (RedVortexDev) [inline src/main/java/dev/dfonline/codeclient/Commands.java:613]: turn the Objects.requireNonNull check you removed and `assert CodeClient.MC.player != null` to a `if (CodeClient.MC.player == null || X) return;` so its cleaner and doesnt throw an error in some weird state where they are null

**Review by RedVortexDev — CHANGES_REQUESTED** [review]

File: src/main/java/dev/dfonline/codeclient/Commands.java (hunk 1 above)
Comment (MrSam7K) [reply src/main/java/dev/dfonline/codeclient/Commands.java:613]: i actually did not see assert there lol

**Review by RedVortexDev — APPROVED** [review]

**Comment by RedVortexDev** [comment]
waiting for any feedback from @GeorgeRNG

**Review by GeorgeRNG — APPROVED** [review]

**Pull request merged**